src/settings/components/general-settings/appearance-settings.tsx

```diff
@@ -43,13 +45,18 @@ const AppearanceSettings: FC<{
 		[darkReaderResultValue, items]
 	);
 	const onSelectionChange = useCallback(
-		(v) => {
-			if (DR_VALUES.includes(v) && v !== darkReaderResultValue) {
-				setDarkReaderState(v);
-				addMod('props', 'zappDarkreaderMode', { app: SHELL_APP_ID, value: v });
+		(value) => {
```

Comment:
value is an implicit any here. We can type the useCallback as `useCallback<NonNullable<SelectProps['onChange']>>`